index.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+    font-family: 'League Spartan';
+    src: url('fonts/subset-LeagueSpartan-Bold.eot');
+    src: url('fonts/subset-LeagueSpartan-Bold.eot?#iefix') format('embedded-opentype'),
+        url('fonts/ssubset-LeagueSpartan-Bold.woff') format('woff'),
+        url('fonts/subset-LeagueSpartan-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+/* Header styling */
+.wrapper {
+
+}
+
+.header {
+	width: 612px;
+	height: 44px;
+    margin: 38px 720px 0 108px;
+    
+}
+
+.list {
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    margin:0;
+}
+
+.header__logo {
+	width: 16px;
+	height: 44px;
+}
```

Comment:
The SVG itself has width and height, so probably these won't make a lot of sense here, but they might do in the `<img>` tag. [Here's why](https://stackoverflow.com/a/1247693/1333836).